cpp/velox/substrait/SubstraitToVeloxExpr.cc

```diff
@@ -163,6 +180,44 @@ VectorPtr constructFlatVector(
   return vector;
 }
 
+TypePtr getScalarType(const ::substrait::Expression::Literal& literal) {
+  auto typeCase = literal.literal_type_case();
+  switch (typeCase) {
+    case ::substrait::Expression_Literal::LiteralTypeCase::kBoolean:
+      return BOOLEAN();
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI8:
+      return TINYINT();
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI16:
+      return SMALLINT();
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI32:
+      return INTEGER();
+    case ::substrait::Expression_Literal::LiteralTypeCase::kFp32:
+      return REAL();
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI64:
+      return BIGINT();
```

Comment:
Maybe exchange the order of BIGINT and REAL to improve readability.

Reply:
Updated, thanks.